app/queries/get_change_offer_options.rb

```diff
@@ -22,7 +22,7 @@ def available_study_modes(course:)
     CourseOption
       .where(course: offerable_courses.find_by(id: course.id))
       .group('study_mode')
-      .pluck(:study_mode)
+      .pluck(:study_mode).sort
```

Comment:
Would `order` work here?

```
CourseOption
  .where(course: offerable_courses.find_by(id: course.id))
  .group(:study_mode)
  .order(:study_mode)
  .pluck(:study_mode)
```

Sorting directly in the database instead of fetching the results and sorting in ruby.

Reply:
Sorting an array that will never have a length greater than 2 is probably faster, but since the overall number of records is so small it makes no real difference. Do you have a strong preference? Any particular reason to do one way and not the other?

Reply:
Nope, no string preference, I know it's very minor, it just looks neater to me. I'm happy to approve.